scripts/genunicodetables.lua

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/lua
```

Comment:
I don't care either way, but I think if we're arguing it doesn't matter what the shebang is because we run it via lua directly, then it doesn't make sense for it to be there at all and for the file to be marked executable.  FWIW, there's only one other script, and it could be changed too (not as part of this PR). :-)

I don't feel strongly either way about it, just pointing out where that line of thinking leads. :-)